pages/index.ejs

```diff
@@ -1,7 +1,6 @@
 ---
 title: Homepage
-description: Get your webiste up and running online in no time using current starter
-header: CMintS starter
+description: CBA Chromium Browser Automation (extension for chrome browser automation) 
```

Comment:
Detail: Seems like description is not being used per page basis, we might eventually introduce description to improve SEO, but as it's not a requirement and as we are using `subtitle` variable specified in the `config.js` to show the sub header we can either remove this or return it to it's initial state.